src/styles/typography/index.md.njk

```diff
@@ -105,7 +105,7 @@ If it's an external link to a non-government website, make that clear in the lin
 
 ### Opening links in a new tab
 
-Avoid opening links in a new tab or window. It can be disorienting - and [can cause accessibility problems for people who aren't able to visually perceive that the new tab has opened](https://www.w3.org/TR/WCAG20-TECHS/G200.html).
+Avoid opening links in a new tab or window. It can be disorienting - and [can cause accessibility problems for people who are not able to visually perceive that the new tab has opened](https://www.w3.org/TR/WCAG20-TECHS/G200.html).
```

Comment:
```suggestion
Avoid opening links in a new tab or window. It can be disorienting - and [can cause accessibility problems for people who cannot visually perceive that the new tab has opened](https://www.w3.org/TR/WCAG20-TECHS/G200.html).
```